src/main/java/pl/isa/biblioteka/books/BorrowBooks.java

```diff
@@ -0,0 +1,84 @@
+package pl.isa.biblioteka.books;
+
+import pl.isa.biblioteka.user.LogUser;
+import pl.isa.biblioteka.user.Person;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class BorrowBooks {
+    public static Scanner scanner = new Scanner(System.in);
+    public static BooksEdit booksEdit = new BooksEdit();
+
+    public static LogUser logUser = new LogUser();
+
+    public static boolean addBookToPerson() {
+        List<Book> booksList = BooksEdit.booksList;
+        System.out.println("Podaj tytuł książki do wypożyczenia: ");
+        String bookTitle = scanner.nextLine();
+        for (Book book : booksList) {
+            if (book.getTitle().equalsIgnoreCase(bookTitle) && book.isState()) {
+                book.setState(false);
+                logUser.getPersonBooks().add(book);
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static boolean returnBook() {
+        List<Book> booksList = BooksEdit.booksList;
+        System.out.println("Podaj tytuł książki do zwrócenia: ");
+        String bookTitleToReturn = scanner.nextLine();
+        for (Book personBook : logUser.getLogPerson().getPersonBooks()) {
+            if (personBook.getTitle().equalsIgnoreCase(bookTitleToReturn) && !personBook.isState()) {
+                for (Book book : booksList) {
+                    if (book.getTitle().equalsIgnoreCase(bookTitleToReturn)) {
+                        book.setState(true);
+                    }
+                }
+                logUser.getPersonBooks().removeIf(foundBookByTitle(bookTitleToReturn));
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static void sortByCategory() {
+        List<Book> books = BooksEdit.booksList.stream().filter(Book::isState).toList();
+        Set<String> availableCategory = showAvailableCategory(books);
+        showFilterBookByCategory(books, availableCategory);
+
+    }
+
+    private static Set<String> showAvailableCategory(List<Book> books) {
```

Comment:
To też powinno robić menu, a BorrowBooks powinien tylko zwracać listę dostępnych kategorii.